src/components/views/elements/DialogButtons.js

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+"use strict";
```

Comment:
This isn't necessary now

Reply:
Is that referring to the `"use scrict"`?  The lines you've highlighted make this a bit unclear.

Reply:
Yes, sorry - `use strict` isn't necessary in a module (they're all implicitly strict).